src/main/java/org/jitsi/videobridge/simulcast/SimulcastStream.java

```diff
@@ -243,4 +243,29 @@ public void askForKeyframe()
     {
         simulcastReceiver.askForKeyframe(this);
     }
+
+    @Override
+    public boolean equals(Object o)
+    {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        SimulcastStream that = (SimulcastStream) o;
+
+        if (primarySSRC != that.primarySSRC) return false;
+        if (rtxSSRC != that.rtxSSRC) return false;
+        if (fecSSRC != that.fecSSRC) return false;
+        return order == that.order;
+
+    }
+
+    @Override
+    public int hashCode()
+    {
+        int result = (int) (primarySSRC ^ (primarySSRC >>> 32));
+        result = 31 * result + (int) (rtxSSRC ^ (rtxSSRC >>> 32));
+        result = 31 * result + (int) (fecSSRC ^ (fecSSRC >>> 32));
+        result = 31 * result + order;
```

Comment:
This makes me nervous. These fields should all be final if they are used in hashCode(). For primarySSRC, fecSSRC and order this can be made easily, the rtxSSRC will require a bit of refactoring.